.eslintrc

```diff
@@ -0,0 +1,220 @@
+{
+    "parser": "@typescript-eslint/parser",
+    "plugins": [
+        "@typescript-eslint",
+        "sonarjs",
+        "security",
+        "github"
+    ],
+    "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/eslint-recommended",
+        "plugin:@typescript-eslint/recommended",
```

Comment:
I don't think so. I check the typescript eslint plugin and they have `@typescript-eslint/recommended` plugin in their README.  

The reason I added those because both of those plugins were also present in `generator-model-sdk` repo.

I will remove the `eslint-recommended` plugin.